### PR TITLE
feat: Added Series Data to card lists

### PR DESCRIPTION
**Pull request by vinnyjth**
## 🐛 Issue

https://3.basecamp.com/3926363/buckets/32694519/card_tables/cards/7019377449

## ✏️ Solution

Add series data to the rendered ContentCards

## 🔬 To Test

1. from web-embeds...
2. `yarn dev` 

The rendered feed should show cards with series data.

## 📸 Screenshots
CleanShot 2024-02-01 at 12 06 54@2x

CleanShot 2024-02-01 at 12 06 06@2x

### CODE REVIEW

File: packages/web-shared/components/FeatureFeed/Features/HorizontalCardListFeature.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ function HorizontalCardListFeature(props = {}) {
     return null;
   }
 
+  console.log(props.feature.cards);
```
Comment (nlewis84) [inline packages/web-shared/components/FeatureFeed/Features/HorizontalCardListFeature.js:86]: nit: log

**Review by nlewis84 — APPROVED** [review]
Looks nice! One nitpick.

File: packages/web-shared/components/FeatureFeed/Features/HorizontalCardListFeature.js (hunk 1 above)
Comment (vinnyjth) [reply packages/web-shared/components/FeatureFeed/Features/HorizontalCardListFeature.js:86]: thanks :D

Comment (vinnyjth) [inline packages/web-shared/components/FeatureFeed/Features/HorizontalCardListFeature.js:86]:
```suggestion
```

**Pull request merged**